.github/workflows/01-build-and-release.yml

```
@@ -4,28 +4,21 @@ on:
   push:
     branches:
       - main
-      - set-up-cicd
+      - use-preinstalled-djgpp-image-in-workflows
 permissions:
   contents: write
 jobs:
   release-job:
-    runs-on: ubuntu-latest
+    runs-on: ghcr.io/volkertb/debian-djgpp:v0.2
```

`runs-on` modifies the runner, you want to switch to a different Docker image:

```suggestion
    runs-on: ubuntu-latest
    container:
      image: ghcr.io/volkertb/debian-djgpp:v0.2
```